src/lib/logger.js

```diff
@@ -22,6 +23,16 @@ function getlvl(x) {
   }
 }
 
+/**
+ * A RotatingFileStream that modifes the message first
+ */
+class VerdaccioRotatingFileStream extends Logger.RotatingFileStream { // We depend on mv so that this is there
```

Comment:
@juanpicado I am sure about extending the `Logger.RotatingFileStream`

Reply:
What do you suggest instead? Reimplementing it is a lot of work and this is public interface so a breaking change would be declarative

Reply:
make sense.